integration-tests/src/lockup_interface.rs

```diff
@@ -21,6 +25,42 @@ pub struct LockupContract<'a> {
     contract: &'a Contract,
 }
 
+impl LockupContract<'_> {
+    pub(crate) async fn terminate_raw(
+        &mut self,
+        lockup_index: LockupIndex,
+        hashed_schedule: Option<Schedule>,
+        termination_timestamp: Option<TimestampSec>,
+    ) -> Result<ExecutionFinalResult> {
+        println!("▶️ terminate");
+
+        let result = self
+            .user_account()
+            .expect("User account is required")
+            .call(self.contract.id(), "terminate")
+            .args_json(json!({
+                "lockup_index": lockup_index,
+                "hashed_schedule": hashed_schedule,
+                "termination_timestamp": termination_timestamp
+            }))
+            .max_gas()
+            .deposit(NearToken::from_yoctonear(1))
+            .transact()
+            .await?;
```

Comment:
I updated integration utils in this repo and you can use new interface for such calls. You can rebase to main and use them